README.md

```diff
@@ -72,56 +72,78 @@ BigchainDbConfigBuilder
 
 ### Example: Prepare keys, assets and metadata
 ```java
-//    prepare your keys
+// prepare your keys
 net.i2p.crypto.eddsa.KeyPairGenerator edDsaKpg = new net.i2p.crypto.eddsa.KeyPairGenerator();
 KeyPair keyPair = edDsaKpg.generateKeyPair();
 
-//    New asset
+// New asset
 Map<String, String> assetData = new TreeMap<String, String>() {{
     put("city", "Berlin, DE");
     put("temperature", "22");
     put("datetime", new Date().toString());
 }};
 
-//    New metadata
+// New metadata
 MetaData metaData = new MetaData();
 metaData.setMetaData("what", "My first BigchainDB transaction");
 ```
 
-### Example: Create a Transaction
-```java	
-//    Set up your transaction
-Transaction transaction = BigchainDbTransactionBuilder
-	.init()
-	.addAssets(assetData, TreeMap.class)
-	.addMetaData(metaData)
-	.operation(Operations.CREATE)
-	.buildOnly((EdDSAPublicKey) keyPair.getPublic());
-```
+### Example: Create an Asset
+
+Performing a CREATE transaction in BigchainDB allocates or issues a digital asset.
 
-### Example: Create and Sign Transaction
 ```java
-//    Set up your transaction
-Transaction transaction = BigchainDbTransactionBuilder
+// Set up, sign, and send your transaction
+Transaction createTransaction = BigchainDbTransactionBuilder
 	.init()
 	.addAssets(assetData, TreeMap.class)
 	.addMetaData(metaData)
 	.operation(Operations.CREATE)
-	.buildAndSignOnly((EdDSAPublicKey) keyPair.getPublic(), (EdDSAPrivateKey) keyPair.getPrivate());
-
+	.buildAndSign((EdDSAPublicKey) keyPair.getPublic(), (EdDSAPrivateKey) keyPair.getPrivate())
+	.sendTransaction();
 ```
 
-### Example: Create, Sign and Send a Transaction
+### Example: Transfer an Asset
+
+Performing a TRANSFER transaction in BigchainDB changes an asset's ownership (or, more accurately, authorized signers):
+
```

Comment:
add an example to the readme of a `TRANSFER` operation. this would have really helped me get started.